mwdb/core/search/fields.py

```diff
@@ -150,11 +151,37 @@ def get_condition(self, expression: Expression, remainder: List[str]) -> Any:
         value = get_term_value(expression)
 
         if expression.has_wildcard():
+            if not self.support_wildcards:
+                raise UnsupportedGrammarException("Wildcards are not allowed here")
             return self.column.any(self.value_column.like(value))
         else:
             return self.column.any(self.value_column == value)
 
 
+class UUIDField(BaseField):
+    def __init__(self, column, value_column):
```

Comment:
types

Reply:
I have problem with types here because constructor accepts sqlalchemy columns. AFAIR sqlalchemy doesn't have anything to give proper typing for them.